kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/ZookeeperConf.scala

```diff
@@ -87,26 +89,34 @@ object ZookeeperConf {
       .intConf
       .createWithDefault(120)
 
-  val ZK_MIN_SESSION_TIMEOUT: OptionalConfigEntry[Int] =
+  val ZK_MIN_SESSION_TIMEOUT: ConfigEntry[Int] =
     buildConf("zookeeper.embedded.min.session.timeout")
       .doc("minSessionTimeout in milliseconds for the embedded zookeeper server will allow the" +
         " client to negotiate. Defaults to 2 times the tickTime")
       .version("1.2.0")
       .intConf
-      .createOptional
+      .createWithDefault(3000 * 2)
 
-  val ZK_MAX_SESSION_TIMEOUT: OptionalConfigEntry[Int] =
+  val ZK_MAX_SESSION_TIMEOUT: ConfigEntry[Int] =
     buildConf("zookeeper.embedded.max.session.timeout")
       .doc("maxSessionTimeout in milliseconds for the embedded zookeeper server will allow the" +
         " client to negotiate. Defaults to 20 times the tickTime")
       .version("1.2.0")
       .intConf
-      .createOptional
+      .createWithDefault(3000 * 20)
 
+  @deprecated(s"the config will be removed", since = "1.3.0")
   val ZK_SERVER_ID: ConfigEntry[Int] =
     buildConf("zookeeper.embedded.server.id")
       .doc("serverId for the embedded zookeeper server")
       .version("1.2.0")
       .intConf
       .createWithDefault(-1)
+
+  val ZK_DELETE_DATA_DIRECTORY_ON_CLOSE: ConfigEntry[Boolean] =
+    buildConf("zookeeper.embedded.delete-data-directory-on-close")
```

Comment:
Can we remove this？and keep it AS-IS？

Reply:
hi, you mean always let `ZK_DELETE_DATA_DIRECTORY_ON_CLOSE` = true ?

Reply:
![image](https://user-images.githubusercontent.com/86483005/128844840-2807fa1f-2a6e-435a-b998-7dcebb93fa88.png)

Reply:
Yes, we currently don't have a proper recovery mechanism  for the embedded zookeeper

Reply:
Got